pkg/cmd/util/env.go

```diff
@@ -31,3 +34,21 @@ func GetEnv(key string) (string, bool) {
 	}
 	return val, true
 }
+
+// ThreadSafeSetEnv provides locking around the call to os.Setenv
+// do an internet search like "golang setenv thread safe" and
```

Comment:
It's technically not just Go, just that environment isn't thread safe period.

Reply:
@stevekuznetsov - based on the POSIX and ISO references I found at https://stackoverflow.com/questions/30476600/how-can-getenv-be-thread-safe and that getenv need not be thread safe, that is why I went with only protecting setenv